docfiles/docs.js

```diff
@@ -189,9 +189,24 @@ function setupSemantic() {
         }).appendTo(outer);
     });
 
-    $('#printbtn').on("click", function () {
-        window.print();
-    })
+    $('#printbtn').dropdown({
+        action: function(text, value) {
```

Comment:
this doesn't seem to activate on space bar, only enter key

Reply:
Is this still the case?

Reply:
Yup, still won't activate on space

Reply:
I took a stab at fixing this but couldn't find a working solution. It's even broken on semantic UI's demo page: https://semantic-ui.com/modules/dropdown.html#/examples
I vote we punt this to another work item